cucumber/authenticators_gce/features/authn_gce.feature

```diff
@@ -33,12 +33,41 @@ Feature: GCE Authenticator - Hosts can authenticate with GCE authenticator
     And I set "authn-gce/service-account-email" annotation to host "test-app"
     And I set "authn-gce/project-id" annotation to host "test-app"
     And I set "authn-gce/instance-name" annotation to host "test-app"
-    And I obtain a GCE identity token in "full" format with audience claim value: "conjur/cucumber/host/test-app"
+    And I obtain a GCE identity token in full format with audience claim value: "conjur/cucumber/host/test-app"
     And I save my place in the audit log file
-    When I authenticate with authn-gce using Google identity token
+    When I authenticate with authn-gce using token and account "cucumber"
```

Comment:
i would use `cucumber` as the default account so you don't need to send it each time. the regex `and account ""` can be optional